docs/configure_open_edx.md

```diff
@@ -0,0 +1,54 @@
+Install Open edX
+---
+
+
+#### Setup Open edX Devstack
+
+Following steps are inspired by [edx-devstack](https://github.com/edx/devstack).
+
+#### Clone edx/devstack
+
+```
+$ git clone https://github.com/edx/devstack
+$ cd devstack
+$ git checkout open-release/ironwood.master
```

Comment:
Since MicroMasters connects to edx.org, it might make more sense to checkout master instead of the Ironwood release. 

of course, that comes with other risks though.

Reply:
Updated